.github/workflows/python-publish.yml

```diff
@@ -1,31 +1,33 @@
 # This workflows will upload a Python Package using Twine when a release is created
-# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+ 
 name: Upload Python Package
-
+ 
```

Comment:
I think we can revert this change. seems wrong

Reply:
@lnxpy have you seen my comment here?

Reply:
@hramezani I had changed the structure once you commented that message. I reverted the change though.

Reply:
Still I can see the change 🤔

Reply:
I think I fixed something else back in the moment. It's fixed now. (IT HAS TO 😆 )